lib/rucio/core/dirac.py

```diff
@@ -51,7 +54,7 @@ def _exists(scope, name, session=None):
 
 
 @transactional_session
-def add_files(lfns, account, ignore_availability, session=None):
+def add_files(lfns, account, ignore_availability, session=None, vo='def'):
```

Comment:
`session` should stay the last argument

Reply:
This is a relatively new code, so assuming no-one is using purely positional arguments here. Will make the change.